e2e-polybft/framework/test-cluster.go

```diff
@@ -481,6 +483,26 @@ func (c *TestCluster) WaitUntil(dur time.Duration, handler func() bool) error {
 		}
 	}
 }
+func (c *TestCluster) WaitForBalance(dur time.Duration, address ethgo.Address) (*big.Int, error) {
+	timer := time.NewTimer(dur)
+	defer timer.Stop()
+
+	for {
+		select {
+		case <-timer.C:
+			return nil, fmt.Errorf("timeout occurred while waiting for balance ")
+		case <-time.After(150 * time.Millisecond):
```

Comment:
Its better to use `ticker` (https://gobyexample.com/tickers) here
Maybe it would be good to change this helper func to have one more argument (currently `c.Servers[0]` is hardcoded and maybe we want to invoke some other server)?
Same goes to balance check (in this case hardcoded 0)

Reply:
new parameter added via: fixed: https://github.com/0xPolygon/polygon-edge/pull/1258/commits/8de4d6a725d75f5819e1071cfac7a5ffbab412fb

Reply:
ticker added: https://github.com/0xPolygon/polygon-edge/pull/1258/commits/72e830ea528acae6bfba2d32d5dd589a796a6c70